config/default/pyproject.toml.j2

```diff
@@ -19,6 +19,21 @@ directory = "bugfix"
 name = "Bug fixes:"
 showcontent = true
 
+[[tool.towncrier.type]]
+directory = "internal"
+name = "Internal:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "doc"
```

Comment:
This should be `documentation`, like in [plone.api](https://github.com/plone/plone.api/blob/master/pyproject.toml), to avoid apps trying to open Word when you click on it.

```suggestion
directory = "documentation"
```

Reply:
@mauritsvanrees Could we use `.docs`? `.documentation` is a bit annoying to type every time...

Reply:
Oh, then we have to fix it in [plone.restapi](https://github.com/plone/plone.restapi/blob/master/pyproject.toml) as I copied it from there 😅

Reply:
`.doc` and `.docs` are extensions used by office software. When opening them, it launches office, even from IDEs, and that is way more annoying than typing a few more letters. Believe me, if I could get away with a shorter extension, I would.

Reply:
@stevepiercy Okay. I knew about that for `.doc`, but didn't realize it was the case for `.docs` as well.